tests/monitors/kubernetes_cluster/kubernetes_cluster_test.py

```diff
@@ -21,6 +21,7 @@ def test_kubernetes_cluster_in_k8s(k8s_cluster):
         SCRIPT_DIR / "resource_quota.yaml",
         TEST_SERVICES_DIR / "nginx/nginx-k8s.yaml",
         SCRIPT_DIR / "cronjob.yaml",
+        SCRIPT_DIR / "statefulset.yaml",
```

Comment:
How did this pass before without this?

Reply:
> How did this pass before without this?

I think we were getting false positives in circleci if the `kubernetes_cluster_test:test_stateful_sets` was running at the same time.  Running the test by itself locally has been failing for me consistently.

Reply:
Ok, makes sense.